opentok/vonage-client-sdk-video
# VIDCS-2071: Privacy manifest file

#### What is this PR doing?
Added a privacy file setting in resources like 
```swift
 .target(name: "VonageClientSDKVideo",
                path: "Sources",
                resources: [
                    .process("VonageClientSDKVideo/Resources/selfie_segmentation.tflite"),
                    .copy("VonageClientSDKVideo/Resources/PrivacyInfo.xcprivacy")
                ],
```

#### How should this be manually tested?
 Pull in the branch  in Xcode "Add package" and Archive the project. 
After that right click on that archive and generate a privacy report and save it as a pdf.
The privacy report should have been generated and you should see the prvacy settings in that file

#### What are the relevant tickets?

[VIDCS-2071](https://jira.vonage.com/browse/VIDCS-2071)
